speechbrain/utils/train_logger.py

```diff
@@ -200,3 +209,19 @@ def log_stats(
             self.run.log({**logs, **stats_meta}, step=step)
         else:
             self.run.log({**logs, **stats_meta})
+
+
+def plot_spectrogram(spectrogram, ap=None, fig_size=(16, 10), output_fig=False):
+    import matplotlib
+
+    matplotlib.use("Agg")
+    import matplotlib.pyplot as plt
```

Comment:
We may want to be careful about using the global `plt` here in a reusable library - it is better to draw on the axis object.